frontend/src/components/dashboard/PremiumPromoBanners.tsx

```diff
@@ -4,15 +4,30 @@ import styles from "./ProfileBanners.module.scss";
 import PhoneIllustration from "./images/phone-premium-promo.svg";
 import { Banner } from "../Banner";
 
-export const PremiumPromoBanners = () => {
+export type Props = {
+  showFirstPremiumBanner?: boolean;
+};
+
+export const PremiumPromoBanners = (props: Props) => {
   const banners: ReactNode[] = [];
 
-  banners.push(<StopSpamBanner key="stop-spam-premium-banner" />);
-  banners.push(
-    <AdvancedIdentityBanner key="advanced-identity-premium-banner" />
-  );
-  banners.push(<ControlReceiverBanner key="control-receiver-premium-banner" />);
-  banners.push(<ExtraProtectionBanner key="extra-protection-premium-banner" />);
+  // Only show the first banner while waiting for the A/B test architecture to be built out
+  {
+    props.showFirstPremiumBanner
+      ? banners.push(<StopSpamBanner key="stop-spam-premium-banner" />)
+      : null;
+  }
+
+  // TODO: Implement A/B architecture to following banners
+  // banners.push(<StopSpamBanner key="stop-spam-premium-banner" />);
+  //   banners.push(
+  //     <AdvancedIdentityBanner key="advanced-identity-premium-banner" />
+  //   );
+  //   banners.push(
+  //     <ControlReceiverBanner key="control-receiver-premium-banner" />
+  //   );
+  //   banners.push(
+  //     <ExtraProtectionBanner key="extra-protection-premium-banner" />);
```

Comment:
Question: do we want to keep this todo comment here? Or would it be better to open a ticket (if one isn't already open) so we don't lose track of it?

Reply:
There is already a ticket for this - since the work is being paused on at the moment. I added a comment about the commented out banners here: https://mozilla-hub.atlassian.net/browse/MPP-2568?jql=text%20~%20%22a%2Fb%20test%20premium%20banner%22